vllm/lora/layers.py

```diff
@@ -1154,6 +1159,11 @@ def _get_logits(
         if logits is None:
             return None
 
+        if self.shared_to_full_mapping_gpu is not None:
+            # Reindex full logits tensor to ensure 1:1 mapping between
```

Comment:
Can you give me a simple example with/without shared_to_full_mapping_gpu indexing in comments?